pkgs/applications/version-management/gogs/default.nix.patch

```diff
@@ -0,0 +1,22 @@
+2,3c2
```

Comment:
@joachifm `default.nix` is generated by go2nix, but it cannot guess the `postInstall` hook required to wrap Gogs (because it's specific to Gogs), nor the optional SQLite support with an argument.

That's why, when auto-generating `default.nix` using `gogs-derivation.nix`, I patch the output of go2nix to include the `sqliteSupport` argument, and `postInstall` hook in order to have the final `default.nix`.

All of this is done inside `nix-build --no-out-link gogs-definition.nix`.